HoneyBadgers.WebApp/Views/Movie/Details.cshtml

```diff
@@ -0,0 +1,90 @@
+﻿@model HoneyBadgers.Logic.Movie
+
+@{
+    ViewData["Title"] = "Details";
+}
+
+<h1>Details</h1>
+
+<div>
+    <h4>Movie</h4>
+    <hr />
+    <dl class="row">
+        <dt class = "col-sm-2">
+            @Html.DisplayNameFor(model => model.Id)
```

Comment:
Powiedz proszę, że przejdziemy w późniejszym czasie na `<label asp-for=""></label>` 😄 wydaje mi się czytelniejsze

Reply:
tak, tak jak najbardziej nie skupiałam się za bardzo nad details, bo to nie do końca było moim zadaniem, a chciałam tylko dodać by sprawdzić czy działa przekierowanie 😉